src/www/ui/api/Controllers/AuthController.php

```diff
@@ -60,6 +60,7 @@ public function optionsVerification($request, $response, $args)
     global $SysConf;
     return $response->withStatus(204)
       ->withHeader('Access-Control-Allow-Origin', $SysConf['SYSCONFIG']['CorsOrigins'])
+      ->withHeader('Access-Control-Expose-Headers', $SysConf['SYSCONFIG']['Headers'])
```

Comment:
I was having discussion with @shaheemazmalmmd. He suggested, and also a recommended practice to send these headers for all requests. So how about moving all the CORS headers and put them at https://github.com/fossology/fossology/blob/1d7f5f9fc8c47781fcd327a46727b1a84789aff6/src/www/ui/api/Middlewares/RestAuthMiddleware.php#L97 instead?

That way, we can leave just the 204 response in this function and all the CORS headers will be available for all the responses from the API. What do you think?

Reply:
Yes, surely we can do that.

Reply:
@GMishx updated, please check does this wok

Reply:
Awesome!